src/EA.Weee.Web/Areas/Producer/ViewModels/OrganisationDetailsTabsViewModel.cs

```diff
@@ -27,6 +28,8 @@ public class OrganisationDetailsTabsViewModel
         public bool IsRegistered => this.Status == SubmissionStatus.Submitted && this.HasPaid;
 
         public bool ShowReturnRegistrationToUser => this.IsInternalAdmin && (IsRegistered || this.Status == SubmissionStatus.Submitted);
+        public bool ShowContinueRegistrationToUser => !this.IsInternalAdmin && this.Status == SubmissionStatus.InComplete && this.ApiDateYear == this.Year;
```

Comment:
@will-hope98 probaby worth adding some unit tests for this property... think there is a similar test for ShowPaymentLink